src/cltk/stops/cop.py

```diff
@@ -0,0 +1,329 @@
+"""
+This list is adapted from https://github.com/computationalstylistics/tidystopwords
+which in turn is based on UniversalDependencies treebanks.
+"""
+
+STOPS = [
+	"ϣⲁ", # lemma: ϣⲁ, UD_pos: ADP
+	"ϣⲁⲣⲟ", # lemma: ϣⲁ, UD_pos: ADP
+	"ϣⲁⲣⲱ", # lemma: ϣⲁ, UD_pos: ADP
+	"ϩⲁ", # lemma: ϩⲁ, UD_pos: ADP
+	"ϩⲁϩⲧⲏ", # lemma: ϩⲁϩⲧⲛ, UD_pos: ADP
+	"ϩⲁⲣⲁⲧ", # lemma: ϩⲁⲣⲁⲧ, UD_pos: ADP
+	"ϩⲁⲣⲟ", # lemma: ϩⲁ, UD_pos: ADP
+	"ϩⲁⲣⲱ", # lemma: ϩⲁ, UD_pos: ADP
+	"ϩⲁⲧⲉ", # lemma: ϩⲁⲧⲉ, UD_pos: ADP
+	"ϩⲁⲧⲉ", # lemma: ϩⲁϩⲧⲛ, UD_pos: ADP
+	"ϩⲁⲧⲛ", # lemma: ϩⲁⲧⲛ, UD_pos: ADP
+	"ϩⲓ", # lemma: ϩⲓ, UD_pos: ADP
+	"ϩⲓϫⲙ", # lemma: ϩⲓϫⲛ, UD_pos: ADP
+	"ϩⲓϫⲛ", # lemma: ϩⲓϫⲛ, UD_pos: ADP
+	"ϩⲓϫⲱ", # lemma: ϩⲓϫⲛ, UD_pos: ADP
+	"ϩⲓⲣⲙ", # lemma: ϩⲓⲣⲛ, UD_pos: ADP
+	"ϩⲓⲧⲙ", # lemma: ϩⲓⲧⲛ, UD_pos: ADP
+	"ϩⲓⲧⲛ", # lemma: ϩⲓⲧⲛ, UD_pos: ADP
+	"ϩⲓⲧⲟⲟⲧ", # lemma: ϩⲓⲧⲟⲟⲧ, UD_pos: ADP
+	"ϩⲓⲧⲟⲟⲧ", # lemma: ϩⲓⲧⲛ, UD_pos: ADP
+	"ϩⲓⲧⲟⲩⲱ", # lemma: ϩⲓⲧⲟⲩⲛ, UD_pos: ADP
+	"ϩⲓⲱ", # lemma: ϩⲓ, UD_pos: ADP
+	"ϩⲓⲱⲱ", # lemma: ϩⲓ, UD_pos: ADP
+	"ϩⲙ", # lemma: ϩⲛ, UD_pos: ADP
+	"ϩⲙ", # lemma: ϩⲙ, UD_pos: ADP
+	"ϩⲛ", # lemma: ϩⲛ, UD_pos: ADP
+	"ϫⲓⲛ", # lemma: ϫⲓⲛ, UD_pos: ADP
+	"ⲁϫⲛ", # lemma: ⲁϫⲛ, UD_pos: ADP
+	"ⲉ", # lemma: ⲉ, UD_pos: ADP
+	"ⲉ", # lemma: ⲉⲣⲉ, UD_pos: ADP
+	"ⲉ", # lemma: ⲉⲣⲉ_ⲛⲧⲟⲟⲩ, UD_pos: ADP
+	"ⲉϩⲟⲩⲉ", # lemma: ⲉϩⲟⲩⲉ, UD_pos: ADP
+	"ⲉϫⲙ", # lemma: ⲉϫⲛ, UD_pos: ADP
+	"ⲉϫⲛ", # lemma: ⲉϫⲛ, UD_pos: ADP
+	"ⲉϫⲱ", # lemma: ⲉϫⲛ, UD_pos: ADP
+	"ⲉⲣⲁⲧ", # lemma: ⲉⲣⲁⲧ, UD_pos: ADP
+	"ⲉⲣⲟ", # lemma: ⲉ, UD_pos: ADP
+	"ⲉⲣⲱ", # lemma: ⲉ, UD_pos: ADP
+	"ⲉⲧⲃⲉ", # lemma: ⲉⲧⲃⲉ, UD_pos: ADP
+	"ⲉⲧⲃⲏⲏⲧ", # lemma: ⲉⲧⲃⲉ, UD_pos: ADP
+	"ⲉⲧⲟⲟⲧ", # lemma: ⲉⲧⲛ, UD_pos: ADP
+	"ⲕⲁⲧⲁ", # lemma: ⲕⲁⲧⲁ, UD_pos: ADP
+	"ⲙ", # lemma: ⲛ, UD_pos: ADP
+	"ⲙ", # lemma: ⲙ, UD_pos: ADP
+	"ⲙⲙⲟ", # lemma: ⲛ, UD_pos: ADP
+	"ⲙⲙⲟ", # lemma: ⲙⲙⲟ, UD_pos: ADP
+	"ⲙⲙⲱ", # lemma: ⲛ, UD_pos: ADP
+	"ⲙⲛ", # lemma: ⲙⲛ, UD_pos: ADP
+	"ⲙⲛⲛⲥⲁ", # lemma: ⲙⲛⲛⲥⲁ, UD_pos: ADP
+	"ⲙⲛⲛⲥⲱ", # lemma: ⲙⲛⲛⲥⲁ, UD_pos: ADP
+	"ⲛ", # lemma: ⲛ, UD_pos: ADP
```

Comment:
I see, similar duplication here. For our immediate purpose of *identifying* stopwords via their inflected form, it is OK. If we trust the lemmatizer from (currently) out of Stanza, someone could experiment with doing the match on lemma instead.